src/controllers/stallController.ts

```diff
@@ -40,6 +62,27 @@ async function indexStall(req: Request, res: Response, next: NextFunction) {
     const stalls = await Stall.findAll({
       include: getStallInclude(),
     });
+
```

Comment:
This may not work but you can try
```typescript
const stalls = await Review.findAll({ 
    attributes: [...], 
    group: ['stallId'], 
    include: Review.associations.Stall
})
res.status(200).json(stalls);
```

Reply:
I tried this does not work. I get an error when trying this :(

Reply:
Ooo, ok!